### ISSUE TITLE
Facebook Chatbot

### ISSUE THREAD

**Issue by DianaHajTPS** [BeautifulTrouble/beautiful-rising#103]
Hi Phillip,

I cannot seem to get Adah to respond. Could you please advise?

Also, many of the participants have been adding her but I don't know who/how we're accepting the friend requests.

**Comment by phillipadsmith** [comment]
Hi @DianaHajTPS: Several months ago we agreed to only launch with Telegram, and hence there's no mention of Facebook on the site. How are these users finding their way to the old Adah account?

**Comment by DianaHajTPS** [comment]
Yup that's true we did agree, until Søren informed me that the Adah bot is up and running and that we can test it with a couple of participants from the evaluation. Last week a couple of participants had problems with Telegram and some others were curious about other platforms such as Fb and Whatsapp. So based on that information I thought why not have them try the FB bot since it's functioning. I asked them to add Adah, but realized it wasn't working when I tried it myself.

Sorry If I've done so without consulting with you, I thought I had missed a conversation in which you gave the green light on Adah in general.

**Comment by phillipadsmith** [comment]
No need to apologize. I was mostly curious how they'd found their way to Adah...

The use of a Facebook account was an experiment and during that time, Facebook launched "official" Messenger bots to the public, which is what we've decided to use instead of a Facebook account. However, that work is not yet complete or tested. 

The same is true for Skype, where official bot resources were released just a short time ago (but after I'd already built an integration with Skype using a regular Skype account).

Once the current Telegram bot is multi-lingual, I'll focus on re-extending it to Messenger, Skype, etc. 

Hope that helps a bit.

**Comment by DianaHajTPS** [comment]
I see, so I shouldn't be directing anyone to Adah just yet correct?

**Comment by phillipadsmith** [comment]
> I see, so I shouldn't be directing anyone to Adah just yet correct?

Never, actually. Adah is a Facebook account just like yours, or mine. Messenger bots are attached to Pages, not people/profiles.
